### DISCUSSION TITLE
Bagaimana cara mengurutkan index date dalam crosstab?

### DISCUSSION THREAD

**Discussion by srenaldd** [teamalgoritma/community#602] [Q&A] [answered]
Saya mengambil data month-year dari df["date"] dengan  dt.strftime('%m-%Y'). Bagaimana caranya supaya saya bisa mengurutkan month year tsb dengan urutan yang benar secara descending?
![Screenshot (4686)](https://user-images.githubusercontent.com/76720981/228122517-67e0db41-1ebd-4f0a-b4c9-7da40d330cec.png)

**Comment by divakartika** [comment, marked as answer]
Dear @srenaldd, 

Untuk mengurutkan suatu dataframe berdasarkan index nya, dapat digunakan method `.sort_index()`. Namun demikian, berdasarkan dataframe yang terbentuk, index akan terurut berdasarkan tanggal, bukan berdasarkan tahun. Untuk mengurutkan berdasarkan tahun terlebih dahulu, dapat digunakan accessor dan method `.dt.to_period('M')` untuk menghasilkan format year-month.

Demikian yang dapat kami sampaikan, jika jawaban ini membantu, silakan klik "Mark as answer" 😃
